AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1357,6 +1357,8 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 belretail.by##.bannerAfterArticle
+decrypt.co##.sc-118zf0s-2
```

Comment:
It looks like class names like this don't change (I've checked also on iPhone)

Reply:
I don't see this class name.

Reply:
<details>
<img src="https://user-images.githubusercontent.com/18536328/81170531-86aac080-8fa3-11ea-983e-fbcecf06419b.png">
</details>

Reply:
ok